crates/bevy_render/src/camera/camera.rs

```diff
@@ -520,6 +526,18 @@ pub fn camera_system<T: CameraProjection + Component>(
 ) {
     let primary_window = primary_window.iter().next();
 
+    // Disable the camera associated to a window that's just been closed
+    for ev in window_close_events.iter() {
+        for (mut camera, _) in &mut cameras {
```

Comment:
Why not use query.get_mut here?

Reply:
There could be multiple camera per window, like for split screen. Also, the window doesn't have a camera entity id. It's the camera that contains a reference to the window.